### PR TITLE
Create a channel for Vietnamese translation

### PR DESCRIPTION
**Pull request by truongnh1992**
Slack channel for Vietnamese localization.
Refer: cncf/foundation#52

Signed-off-by: Nguyen Hai Truong <truongnh@vn.fujitsu.com>

### CODE REVIEW

**Comment by remyleone** [comment]
👍

File: communication/slack-config/sig-docs/docs-channels.yaml [hunk 1]
```diff
@@ -10,3 +10,4 @@ channels:
   - name: kubernetes-docs-ko
   - name: kubernetes-docs-pt
   - name: kubernetes-docs-zh
+  - name: kubernetes-docs-vi
```
Comment (nikhita) [inline communication/slack-config/sig-docs/docs-channels.yaml]: This needs to be alphabetically sorted :)

Comment (truongnh1992) [reply communication/slack-config/sig-docs/docs-channels.yaml]: Oh, thank @nikhita

**Comment by remyleone** [comment]
@nikhita Could it be merged?

**Comment by mrbobbytables** [comment]
@remyleone we delegate the channel requests in the directories to their associated sigs -- Will loop in some of additional leads for visibility.
/assign @jimangel @zacharysarah

**Pull request merged**